src/test/java/org/opensearch/knn/index/NestedSearchIT.java

```diff
@@ -78,6 +78,30 @@ public void testNestedSearch_whenKIsTwo_thenReturnTwoResults() {
         assertEquals(2, hits.size());
     }
 
+    @SneakyThrows
+    public void testNestedSearchWithFaiss_whenKIsTwo_thenReturnTwoResults() {
```

Comment:
What does KI mean again? Also, this tests the functionality of parent filter, correct?

Reply:
`when_k_is_two`